qt/build.xml

```diff
@@ -351,11 +322,5 @@ limitations under the License.
             </exec>
         </sequential>
     </target>
-		
-	<!-- Should not run the tests from an automated build, because qmlscene doesn't exit at the end. They should
-	     only be run by hand. -->
 	<target name="test" description="Run all tests and build all reports" />
-
-	<target name="doc" />
```

Comment:
The top-level build.xml has a "doc" target which iterates through all the subdirectories and calls "doc" there. If this target does not exist, that target will fail at the top level. Since there are no docs here, it is implemented as nothing, but it at least has to be here so the overall build doesn't fail.

Reply:
oh.. I didn't know that. you're right. So I'll put that code back.